tests/PHPStan/Analyser/data/countable.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Countable;
+
+use function PHPStan\Testing\assertType;
+
+class Foo implements \Countable {
+	public function count() : int {
+		return 0;
+	}
+}
+
+$foo = new Foo();
+assertType('int<0, max>', $foo->count());
```

Comment:
Please put this in an anonymous function. If you leave it like this, the code is actually executed when the tests run (because the class `Foo` is autoloaded).